docs/requirements.txt

```diff
@@ -0,0 +1,11 @@
+sphinx==7.2.6
```

Comment:
I believe doc requirements are in a different requirements.txt and need to be updated there: https://github.com/pytorch/executorch/blob/main/.ci/docker/requirements-ci.txt#L18